### ISSUE TITLE
VIP-Management: Not able to change name and VIP to expire never

### ISSUE THREAD

**Issue by LordofAgents** [MarechJ/hll_rcon_tool#162]
Good day all,
thank you for this amazing piece of software.

Since the last update, we've got the following issue with VIP-management:
We usually give users VIP through the settings-panel to be able to manually input the name of the player. We need to manually change the name in the list to document the origin of the player (like Clan, Group etc.), so we do not use the History -> Player GUI. Since the update (which introduced expiring VIPs), it is not possible to set the expiry to never in the settings menu. If I give a Player VIP through History -> Player, i can set the expiry to never, but I can't change the nickname.

Can you please make it possible to set the expiry to never and change the name?
If possible, it would be cool and user-friendly to be able to do both in both GUIs (History -> Player and Settings).

Thank you!

LordofAgents

TLDR: Give VIP through History > Player, not able to change name. Give VIP through Settings, not able to set Expiry to never.
